src/mardi_importer/integrator/misc.py

```diff
@@ -0,0 +1,16 @@
+def get_wikidata_information(wikidata_id, languages):
```

Comment:
Do we need this file? Isn't this just the same function that is already inside the integrator? If so, I would delete it.

Reply:
removed